<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Rev-PF1JT47A" name="com.revature.tests.dal.PersonDAOTests" tests="3" failures="0" timestamp="2022-04-26T10:49:10 EDT" time="0.005" errors="0">
  <testcase name="getPersonSuccess" time="0.000" classname="com.revature.tests.dal.PersonDAOTests"/>
  <system-out/>
  <testcase name="personDoesNotExist" time="0.004" classname="com.revature.tests.dal.PersonDAOTests"/>
  <system-out/>
  <testcase name="createPersonSuccess" time="0.001" classname="com.revature.tests.dal.PersonDAOTests"/>
  <system-out/>
</testsuite> <!-- com.revature.tests.dal.PersonDAOTests -->
